docs/knowledge/koor/koor-operator.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Koor Operator"
+---
+
+An easy way to install the Koor Storage Distro is by using the [Koor Operator](https://github.com/koor-tech/koor-operator). 
+Using the Koor Operator makes installing, maintaining and upgrading your KSD installation easier. 
+This helps you install a full-fledged Koor Storage Distro (with all Rook-Ceph artifacts) in just 2 lines of helm install.
+
+## Features
+These are the features currently available in the Koor Operator:
+* Install KSD with sane defaults
```

Comment:
Try - instead of * to make this a bullet list. Or maybe you need a blank line before the list.